### ISSUE TITLE
Remove Cherenkov threshold for LS detectors

### ISSUE THREAD

**Issue by JostMigenda** [JostMigenda/sntools#52]
_[based on discussions with Yixuan Jiang]_

Some channels (most importantly `es`, where the contribution at low energies is strongest) currently enforce a Cherenkov threshold on the outgoing particle. While this makes sense for water Cherenkov detectors (to avoid simulating undetectable events), we should remove this threshold for LS detectors.

**Comment by mcolomerm** [comment]
Just to be updated: was this done in the end?

**Comment by JostMigenda** [comment]
No-one got around to this yet, no. Would you (or a student of yours) be interested in doing this? Otherwise I can probably take a look some time next week.

**Comment by mcolomerm** [comment]
OK, we gave a try at directly changing the cherenkov threshold to 0.1, instead of 0.77. But this gives an error as this energy is smaller than the electron mass for the computation of the kinematic bounds as it is implemented in es.py . We can start giving a deeper look: since you know the code, if you have a suggestion at which level is better to put this constrain or remove for LS, let us know as we start by there.

**Comment by JostMigenda** [comment]
Yep—the 0.77 MeV are the _total_ energy of the electron, not just the kinetic energy. (Calculated from $Ε_\text{tot} = \gamma m_0 c^2$ and $\gamma = 1 / (1-\beta^2) = 1/ (1 - 1/n^2)$ where $n=1.33$ is the refractive index of water.) So for LS, you’d have to set this to the electron rest mass.

I’ll look into it this afternoon.
